alphabetical_order.py

```diff
@@ -0,0 +1,9 @@
+##3-alphabetical_order.py
+
+def words ():
+    x, y, z, w = input("please enter four word:").split("-")
+    list=[x, y, z, w]
+    list.sort()
+    for i in list:
+        print(i,end="-")
+print(words())
```

Comment:
cok guzel fakat fonksiyonu cagrisini print fonksiyonu icinde yazdiginiz icin none degeri yazdiriyor ensonda. son satir sadece: words() yeterli. Ancak bu seferde son kelimenin sonuna - isareti yazar. onuda biraz daha dusunseniz temizlenir. mesela for loop icine if i==list[-1]: gibi bir sart koyarak yada baska yollar bulunabilir